devtools/travis-ci/test_format_code.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-CHANGED_FILES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep .py$`
+CHANGED_FILES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep .py$ | grep -v contrib/`
```

Comment:
Wouldn't this select for only files ending in .py that are in contrib? (The opposite of command we want)